stdlib/3/unicodedata.pyi

```diff
@@ -1,37 +1,36 @@
 # Stubs for unicodedata (Python 3.4)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
-
 from typing import Any
 
 ucd_3_2_0 = ...  # type: Any
 ucnhash_CAPI = ...  # type: Any
 unidata_version = ...  # type: str
 
-def bidirectional(unichr): ...
-def category(unichr): ...
-def combining(unichr): ...
-def decimal(chr, default=...): ...
-def decomposition(unichr): ...
-def digit(chr, default=...): ...
-def east_asian_width(unichr): ...
-def lookup(name): ...
-def mirrored(unichr): ...
-def name(chr, default=...): ...
-def normalize(form, unistr): ...
-def numeric(chr, default=...): ...
+_unicode = str
+
+def bidirectional(unichr: _unicode) -> str: ...
+def category(unichr: _unicode) -> str: ...
+def combining(unichr: _unicode) -> int: ...
+def decimal(chr: str) -> int: ...
```

Comment:
Oh yes, they're actually positional-only for some reason. I think the typeshed convention is to start argument names with double underscores to indicate that they're positional-only.